### DISCUSSION TITLE
Zksync Plugin

### DISCUSSION THREAD

**Discussion by thegreatfeez** [Cyfrin/foundry-full-course-cu#2899] [Q&A]
Can’t find the Zksync in my plugin manage, just got to the part of course where i should add zksync
<img width="1433" alt="Screen Shot 2024-11-21 at 5 28 47 PM" src="https://github.com/user-attachments/assets/77bbc2bd-ded0-450a-b138-2f1f9a6a2ffb">

**Comment by EngrPips** [comment]
This sometimes happens due to an unstable internet connection. Confirm that your internet connection is stable

**Reply by chainflash** [reply]
Yes, that's right 💯
